### PR TITLE
allow upload of audio files

### PR DESCRIPTION
**Pull request by camallen**
related to recent changes in the front end https://github.com/zooniverse/Panoptes-Front-End/pull/3863. Seems browsers / systems may detect them differently, e.g.

test_file.mp3: 
file cmd, terminal: audio/mpeg
chrome browser file.type: audio/mp3

test_file.m4a: 
file cmd, terminal: audio/mp4
chrome browser file.type: audio/x-m4a

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: spec/models/medium_spec.rb [hunk 1]
```diff
@@ -15,9 +15,13 @@
 
   describe "#content_type" do
 
-    it 'should not be valid without a valid content_type' do
-      m = build(:medium, content_type: "video/mp4")
-      expect(m).to_not be_valid
+    it 'should not be valid without an invalid content_type' do
```
Comment (zwolf) [inline spec/models/medium_spec.rb]: It should not be valid "_with_ an invalid" (or "without a valid") content type

File: spec/models/medium_spec.rb [hunk 2]
```diff
@@ -26,15 +30,17 @@
     end
 
     it 'should be valid with a valid content_type' do
```
Comment (zwolf) [inline spec/models/medium_spec.rb]: Do you wanna aggregate these failures too, since there's about 8 tests happening in here?

**Comment by camallen** [comment]
requested changes added

**Comment by zwolf** [comment]
LGTM

**Pull request merged**